apiserver/plane/db/models/issue.py

```diff
@@ -95,6 +96,13 @@ class Issue(ProjectBaseModel):
         through="IssueAssignee",
         through_fields=("issue", "assignee"),
     )
+    mentions = models.ManyToManyField(
```

Comment:
Let's not keep a m2m here. Let's only keep the bridge table and user reverse_accessor for filtering